Sources/Shared/VimeoClient.swift

```diff
@@ -156,11 +162,40 @@ final public class VimeoClient {
             return self.perform(
                 request,
                 completionQueue: completionQueue,
+                startImmediatelly: startImmediatelly,
                 then: completion
             )
         }
     }
-    
+
+    /// Executes a `Request` encapsulated into an `EndpointType` and bound to a `Decodable` response.
+    ///
+    /// - Parameters:
+    ///   - endpoint: `EndpointType` object containing the information required to build a request
+    ///   - completionQueue: dispatch queue on which to execute the callback closure
+    ///   - startImmediatelly: a boolean indicating whether or not the request should resume immediately
+    ///   - callback: a closure executed once the request completes, containing a `Result` type
+    ///   for the specified decodable.
+    ///
+    /// - Returns: a `RequestToken` for the in-flight request
+    public func request<Model: Decodable>(
+        _ endpoint: EndpointType,
+        completionQueue: DispatchQueue = .main,
+        startImmediatelly: Bool = true,
```

Comment:
What do you think about swapping these parameters?

Reply:
I don't mind either way, happy to get it swapped.